addons/resource/tests/test_filter_map_domain.py

```diff
@@ -0,0 +1,463 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
Nitpick. You can mvoe this to test_resource module, so that all utils are tested together.